### PR TITLE
ROECCT-125: Add statements to the page

### PR DESCRIPTION
**Pull request by jnicholson-ch**
### JIRA link

https://companieshouse.atlassian.net/browse/ROECCT-125

### Change description

Adds checkboxes with validation to ensure either any of the first 3 checkboxes or last checkbox is checked.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/update/update-layout.html [hunk 1]
```diff
@@ -98,7 +98,7 @@
 {% endblock %}
 
 {% block bodyEnd %}
-  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.js"></script>
+  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v4.6.0/govuk-frontend-4.6.0.min.js"></script>
```
Comment (ConnorOS-ch) [inline views/update/update-layout.html:101]: I think we should probably do this update in a separate PR. I think we have a ticket for it. There are a couple of other places to replace with 4.6: the css on this page and the js and css on `layout.html`

Toro Loco also want us to update the package in `packages.json` to 4.6 then run nmp -i and commit the changes. I think this deserves it's own PR... also makes it easier from a testing perspective

**Review by ConnorOS-ch — CHANGES_REQUESTED** [review]
Looks good, but validation is handled in a separate middleware and uses the express-validator library so would be good to follow the existing pattern

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by jnicholson-ch** [comment]
Closing this PR, the updated version of this PR will be here: https://github.com/companieshouse/overseas-entities-web/pull/1401

**Pull request closed without merging**